lms/envs/bok_choy.py

```diff
@@ -232,16 +232,7 @@
         'course_author': 'http://edx.readthedocs.io/projects/edx-partner-course-staff',
     }
 
-# TODO: TNL-6546: Remove this waffle and flag code.
-from django.db.utils import ProgrammingError
-from waffle.models import Flag
-try:
-    flag, created = Flag.objects.get_or_create(name='unified_course_view')
-    WAFFLE_OVERRIDE = True
-except ProgrammingError:
-    # during initial reset_db, the table for the flag doesn't yet exist.
-    pass
-
+WAFFLE_OVERRIDE = True
```

Comment:
Cool, thanks @robrap. This is the only block I had to touch to get tests to pass, so all the other TODOs will remain unspoilt.